### ISSUE TITLE
Idea for making color accessibility better

### ISSUE THREAD

**Issue by clarfonthey** [tobspr/shapez.io#852]
Hovering over paint to show what color it is is useful, but it may be helpful to use patterns over the sprites to indicate color as well as the actual color of things.

For example, if you have three angles of lines intersecting at 120°, you can use these to indicate the color mixing, as shown in this quickly done image I sketched on my phone:

![sketch](https://user-images.githubusercontent.com/15850505/95695880-e36f6e80-0c06-11eb-875e-a6cf4b7959a7.png)

Bonus is that this is relatively unintrusive so it could be easily done by default and without any extra settings.

**Comment by Hyperion-21** [comment]
Lines would be hard to see because of how small they have to be. This would only work when zoomed in I would think. On the map, resources could have the lines on them.

**Comment by santa-klaus** [comment]
A similar suggestion is already in the [trello](https://trello.com/c/Y5wXoi1y).

**Comment by clarfonthey** [comment]
Yeah, I had only searched the repo before I posted this. I think that both ideas could would be nice to explore. :)
